.changeset/twelve-bananas-kiss.md

```diff
@@ -0,0 +1,7 @@
+---
+"@utrecht/textbox-css": major
```

Comment:
```suggestion
"@utrecht/textbox-css": minor
```

Als er geen breaking change is, dan niet major; als het een feature is dan `minor`